### ISSUE TITLE
Der Vortrag war toll, bis ihr Werbung für GitHub (Microsoft) gemacht habt.

### ISSUE THREAD

**Issue by schrmh** [netzpolitikorg/xandr_recherche_EU#1]
Warum nicht stattdessen beispielsweise für Codeberg e.V. (oder eine eigene Instanz)?

Der erste öffentliche Commit des netzpolitikorg-Accounts auf GitHub ist anscheinend erst rund 5 Monate her....

**Comment by netzpolitikorg** [comment]
Ja, alles gute Fragen und Argumente! Codeberg ist sehr gut, und im Zuge der Überarbeitung unserer eigenen Infrastruktur werden wir mir Sicherheit auch nochmal diskutieren wie und wo wir unseren Code hosten!
